sig-scalability/annual-report-2022.md

```diff
@@ -4,78 +4,95 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - We helped validating scalability and reliability impact of many features across whole year.
+   - We extended scalability testing framework and built a number of new optional tests to
+     cover other usecases (e.g. batch workloads)
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
-
-
+   - Localized scalability/performance improvements across the codebase, e.g.:
+      - Improved LIST calls performance for rare selectors
+      - Optimizing kube-apiserver compression
+      - Optimizing watch by reducing number of allocations
+      - Improved kube-apiserver graceful shutdown
+   - All the improvements to scalability test frameworks in [perf-tests]
 
 3. KEP work in 2022 (v1.24, v1.25, v1.26):
 
+   - SIG Scalability doesn't own non-test code, so all below are officially
+     tracked by other SIGs, but are driven (or co-driven) by SIG Scalability.
+   - Stable
+     - [1164 - Deprecate and Remove SelfLink](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1164-remove-selflink) - 1.24
+     - [1904 - Efficient Watch Resumption](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1904-efficient-watch-resumption) - 1.24
+   - Beta
+     - [1040 - Priority and Fairness for API Server Requests](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1040-priority-and-fairness/README.md) - 1.24, 1.25, 1.26
+     - [1669 - Proxy Terminating Endpoints](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/1669-proxy-terminating-endpoints/README.md) - 1.26
+     - [3453 - Minimize IPtables Restore](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/3453-minimize-iptables-restore)
+   - Alpha
+   - Pre-alpha
+     - [3157 - Watch List](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/3157-watch-list) - 1.26
+
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - Each subproject could benefit from additional hands. However, the Scalability Test Frameworks and Scalability and
+     Performance tests and validation are the ones where we can grow contributors - see [perf-tests] reposityory.
```

Comment:
The paragraph is hard to read, maybe:
"are the ones with the lower barrier to entry"
"repository"
"Other areas require"

Reply:
Done

Reply:
Thanks, also:
"Each Scalability subproject" as currently it reads as any K8s project
"Other areas" not others